bin/sapconf

```diff
@@ -167,7 +167,7 @@ case "$1" in
       exit 1
     fi
     profile=${profile:-sapconf-netweaver}
```

Comment:
we need to change line 165 to check only for 'sapconf' profiles
if [[ $profile != "" && $profile == sapconf-* ]]; then
because only then a 'stop' action makes sense.

and we need to change the profile setting to an absolute setting
profile=sapconf-netweaver
because the use of 'sapconf <profile>' is exactly to set a defined 'sapconf' profile.